requirements.txt

```diff
@@ -1,111 +1,13 @@
-absl-py==0.2.2
-appnope==0.1.0
-asn1crypto==0.24.0
-astor==0.6.2
-astroid==1.6.5
-attrs==18.1.0
-Automat==0.7.0
-backcall==0.1.0
-beautifulsoup4==4.6.0
-bleach==1.5.0
-cachetools==2.1.0
-certifi==2018.4.16
-cffi==1.11.5
-chardet==3.0.4
-click==6.7
-constantly==15.1.0
-cryptography==2.3
-cssselect==1.0.3
-cycler==0.10.0
-decorator==4.3.0
-entrypoints==0.2.3
-Flask==1.0.2
-gast==0.2.0
-google-api-python-client==1.7.4
-google-auth==1.5.0
-google-auth-httplib2==0.0.3
-grpcio==1.12.1
-gspread==3.0.1
-gunicorn==19.9.0
-html5lib==0.9999999
-httplib2==0.11.3
-hyperlink==18.0.0
-idna==2.7
-incremental==17.5.0
-ipykernel==4.8.2
-ipympl==0.2.1
-ipython==6.4.0
-ipython-genutils==0.2.0
-ipywidgets==7.2.1
-isort==4.3.4
-itsdangerous==0.24
-jedi==0.12.0
-Jinja2==2.10
-jsonschema==2.6.0
-jupyter-client==5.2.3
-jupyter-core==4.4.0
-jupyterlab==0.32.1
-jupyterlab-launcher==0.10.5
-kiwisolver==1.0.1
-lazy-object-proxy==1.3.1
-lxml==4.2.1
-Markdown==2.6.11
-MarkupSafe==1.0
-matplotlib==2.2.2
-mccabe==0.6.1
-mistune==0.8.3
-nbconvert==5.3.1
-nbformat==4.4.0
-notebook==5.5.0
-numpy==1.14.5
-oauth2client==4.1.2
-pandas==0.23.1
-pandocfilters==1.4.2
-parsel==1.4.0
-parso==0.2.1
-pexpect==4.6.0
-pickleshare==0.7.4
--e git+git@github.com:BradleyGrantham/fifa-stats-scraper.git@f290c07d70dee10cfc52bd5c3c98e590fe16dcb2#egg=pl_predictions_using_fifa
-prompt-toolkit==1.0.15
-protobuf==3.6.0
-ptyprocess==0.5.2
-pyasn1==0.4.3
-pyasn1-modules==0.2.1
-pycparser==2.18
-PyDispatcher==2.0.5
-Pygments==2.2.0
-pylint==1.9.2
-pyOpenSSL==18.0.0
-pyparsing==2.2.0
-python-dateutil==2.7.3
-python-Levenshtein==0.12.0
-python-slugify==1.2.5
-pytz==2018.4
-pyzmq==17.0.0
-queuelib==1.5.0
-requests==2.19.1
-rsa==3.4.2
-Scrapy==1.5.0
-Send2Trash==1.5.0
-service-identity==17.0.0
-simplegeneric==0.8.1
-six==1.11.0
-tensorboard==1.8.0
-tensorflow==1.8.0
-termcolor==1.1.0
-terminado==0.8.1
-testpath==0.3.1
-tornado==5.0.2
-tqdm==4.23.4
-traitlets==4.3.2
-Twisted==18.4.0
-Unidecode==1.0.22
-uritemplate==3.0.0
-urllib3==1.23
-w3lib==1.19.0
-wcwidth==0.1.7
-webencodings==0.5.1
-Werkzeug==0.14.1
-widgetsnbextension==3.2.1
-wrapt==1.10.11
-zope.interface==4.5.0
+matplotlib==3.1.0
+numpy==1.16.4
+pandas==0.24.2
+Scrapy==1.6.0
+tensorboard==1.14.0
+tensorflow==1.14.0
+Twisted==19.2.1
+pywin32==224
```

Comment:
Where is this used?

Reply:
I tried tensorflow 1.14, but not now I'm back to 1.8 because of some issues I was running into. With pywin32, I'm not sure anymore, but probably because I'm developing on Windows.